homepage/index.html

```diff
@@ -0,0 +1,214 @@
+        </div>
+        <div class="wrapper">
+            <div class="community">
+                <img src="/homepage/images/communitylearning.png" alt="">
+                <span>COMMUNITY LEARNING</span>
+                <p>At access, you can learn with like-minded students who are eager to learn in an immersive environment
+                    just like you. Become part of a community of extremely passionate alumni.</p>
+            </div>
+            <div class="career">
+                <img src="/homepage/images/book.png" alt="">
+                <span>CAREER-FOCUSED CURRICULUM</span>
+                <p>You can learn what you need to know to start a career in the tech industry with our career-focused
+                    curriculum, par with industry standards.</p>
+            </div>
+            <div class="real">
+                <img src="/homepage/images/globe.png" alt="">
+                <span>REAL-WORLD SKILLS</span>
+                <p>At access, you will learn to answer the problems by developing solutions for real - time problems in
+                    the
+                    world. Also, you shall have a chance to interact with industry experts as well.</p>
+            </div>
+        </div>
+
+    </div>
+
+    <div class="course-layout">
+        <div class="layout1">
+            <span>Request A Course Layout</span>
+            <p>Go through our Course Layout, and see what we have to offer you. In the Course Layout, we explore:</p>
+            <div class="bullets">
+                <ul>
+                    <li> Curriculum, programming languages, and technology covered</li>
+                    <li> Daily & Weekly Schedule</li>
+                    <li> Fee Structure</li>
+                    <li> Career Support</li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="layout2">
+            <form action="">
+                <input type="text" id="fullname" name="fullname" placeholder="FULL NAME">
+                <input type="tel" id="phone" name="phone" placeholder="PHONE">
+                <input type="email" id="emailid" name="emailid" placeholder="EMAIL ID">
+                <select name="course" id="course" required>
+                    <option value="" disabled selected hidden>COURSE</option>
+                    <option value="ACCESS 101">ACCESS 101</option>
+                    <option value="ACCESS 102">ACCESS 102</option>
+                    <option value="ACCESS 103">ACCESS 103</option>
+                </select>
+            </form>
+            <button>
+                <p>Send me the Layout</p>
+            </button>
+        </div>
+    </div>
+
+    <div class="online">
+        <div class="flex1">
+            <span>Online or Offline?</span>
+            <p>The only difference between our Online and Offline classes is where you’re located. Both options provide:
+            </p>
+            <ul>
+                <li>Live, Real-Time Instruction</li>
```

Comment:
Bullet points should be having per design please check the design once

<img width="316" alt="Screenshot 2021-09-17 at 8 41 19 AM" src="https://user-images.githubusercontent.com/47788183/133718187-853e64f3-3465-4711-ba08-f2fe688bc3bb.png">

Reply:
done